src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -34,8 +34,9 @@
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
                         <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                                    Name="ContextMenuPanel">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding ElementName=ContextMenuPanel, Path=ActualWidth}" />
```

Comment:
But from the look of it width of `InCanvasSearchControl` is always the same as `ContextMenuPanel`, if we use `Grid`, that can be done without binding at all, right?